backend/django/core/views/api_admin.py

```diff
@@ -362,15 +362,15 @@ def get_project_status(request, project_pk):
     )
     unlabeled_data_objs = list(
         project.data_set.filter(datalabel__isnull=True)
-        .exclude(id__in=stuff_in_queue)
         .exclude(id__in=stuff_in_irrlog)
-        .exclude(id__in=recycle_ids)
```

Comment:
Is there a reason we are including recycled items in the unlabeled count? I think that might get confusing since we explicitly have a recycled count. In addition, if we include recycled items here, then they are subtracted twice in the "other-labeled" calculation